netty/src/test/java/io/grpc/netty/NettyClientTransportTest.java

```diff
@@ -308,11 +329,21 @@ private NettyClientTransport newTransport(ProtocolNegotiator negotiator) {
         DEFAULT_MAX_MESSAGE_SIZE, GrpcUtil.DEFAULT_MAX_HEADER_LIST_SIZE, null /* user agent */);
   }
 
-  private NettyClientTransport newTransport(ProtocolNegotiator negotiator,
-      int maxMsgSize, int maxHeaderListSize, String userAgent) {
-    NettyClientTransport transport = new NettyClientTransport(address, NioSocketChannel.class,
-        group, negotiator, DEFAULT_WINDOW_SIZE, maxMsgSize, maxHeaderListSize, authority,
-        userAgent);
+  private NettyClientTransport newTransport(
+      ProtocolNegotiator negotiator, int maxMsgSize, int maxHeaderListSize, String userAgent) {
+    NettyClientTransport transport = new NettyClientTransport(
+        address, NioSocketChannel.class, new HashMap<ChannelOption<?>, Object>(), group, negotiator,
+        DEFAULT_WINDOW_SIZE, maxMsgSize, maxHeaderListSize, authority, userAgent);
+    transports.add(transport);
+    return transport;
+  }
+
+  private NettyClientTransport newTransport(Map<ChannelOption<?>, ?> channelOptions)
```

Comment:
If this is only used one place, it may be a good idea to just inline it. It's really easy to accumulate a ton of simple helpers like this that end up making the tests harder, not easier.